### PR TITLE
Simplify EventTargetMixin

### PR DESCRIPTION
**Pull request by juanjoDiaz**

### CODE REVIEW

**Comment by juanjoDiaz** [comment]
Broke it into 2 lines so it's shorter 🙂

File: core/util/eventtarget.js [hunk 1]
```diff
@@ -8,33 +8,28 @@
...
         if (!this._listeners.has(type)) {
             this._listeners.set(type, new Set());
         }
         this._listeners.get(type).add(callback);
     }
 
     removeEventListener(type, callback) {
-        if (!this._listeners || !this._listeners.has(type)) {
-            return;
+        if (this._listeners.has(type)) {
```
Comment (CendioOssman) [inline core/util/eventtarget.js]: Very minor nitpick, but is it worth having symmetry with `dispatchEvent()`?

Comment (juanjoDiaz) [reply core/util/eventtarget.js]: The difference is that dispatch returns a boolean and this doesn't return anything.

That would mean:
```javascript
        if (!this._listeners.has(type)) {
            return;
        }

        this._listeners.get(type).delete(callback);
```

which feels a bit unnecessary to me.

But I can change it if you would prefer it 🙂

Comment (samhed) [reply core/util/eventtarget.js]: I like it the way it is, the functions are different enough.

**Pull request merged**